### ISSUE TITLE
JFXTimePicker's UI do not have a 24-hour time format

### ISSUE THREAD

**Issue by Tybobobo** [jfoenixadmin/JFoenix#266]
It does not appear that JFXTimePicker have an 24-hour selection UI.
http://i.imgur.com/PjOGtbD.png

A lot of users would benefit from this, as majority of the world do not use the AM/PM (at least according to Microsoft's Localization). 
https://i.stack.imgur.com/43dwH.jpg

--- 
I would not mind giving it ago myself, but I am not very familiar with the project (yet), and have more pressing matters to attend to. There will come a time when I really need this, and will probably end up making it myself (unless it is already implemented by then) - but until that time, I hope this Issue will give it some attention.

Speaking of which; why not run a IRC or Discord server where users and developers can get in touch and exchange information and knowledge with the use of JFoenix and JavaFX in general? :)

**Comment by jfoenixadmin** [comment]
Hello, 
Thanks for pointing out the issue 👍 , this might be implemented in the future (not planned yet).
in the mean time, feel free to add your own contribution to JFoenix.

Regards,

**Comment by Iw1s** [comment]
As I'm also in need for an 24-hour selection UI for the JFXTimePicker, I decided to give it a shot.

![timepicker](https://cloud.githubusercontent.com/assets/13868083/25278002/45b58dd0-26a1-11e7-9edc-918cddd50204.png) ![timepicker](https://cloud.githubusercontent.com/assets/13868083/25277670/edcebb56-269f-11e7-8384-d8b7f0e73fd5.gif)  

The 24-hour UI can be applied by calling `setIs24HourView(true)` on a `JFXTimePicker` instance.  
To make this possible, changes in `JFXTimePicker.java` and `JFXTimePickerContent.java` were necessary. All of this can be seen in the pull request (#297) I just made.

Accedentially I fucked up `JFXTimePickerContent.java` regarding the formatting. As it was late at night I used my formatting configuration from IntelliJ IDEA on the file before using my brain... 😟 
Feel free to adjust that as needed.

**_Also, keep up the good work. I love your project!_** 💓
